calendarsubscription/store.go

```diff
@@ -128,7 +130,7 @@ func (s *Store) Authorize(ctx context.Context, token string) (context.Context, e
 	}
 
 	var userID string
-	err = s.authUser.QueryRowContext(ctx, c.Subject).Scan(&userID)
+	err = s.authUser.QueryRowContext(ctx, c.Subject, c.IssuedAt).Scan(&userID)
```

Comment:
Do we need to cast this to `time.Time` first?
```suggestion
	err = s.authUser.QueryRowContext(ctx, c.Subject, time.Unix(c.IssuedAt, 0)).Scan(&userID)
```